README.md

```diff
@@ -1,3 +1,20 @@
+About `vim-hack`
+================
+
+`vim-hack` provides file type detection and syntax highlighting for Hack files.
+
+For advanced features, we recommend using this plugin in combination with a
+Language Server Protocol plugin for Vim, such as
+[LanguageClient-neovim], configured to run `hh_client lsp` for `hack` files - for example:
```

Comment:
I _think_ you still need `(LanguageClient-neovim)` after; am I just bad at markdown?

Reply:
Not needed - confirmed with the 'rich diff' thingy in view changes.

If you want to go blind: https://github.github.com/gfm/#shortcut-reference-link